main.py

```diff
@@ -1,17 +1,271 @@
 """
```

Comment:
Note: Regarding naming convention for PR's, it's good practice to name the PR relevant to the task you are working on. A PR is different from a commit so "First WARG Commit" isn't a great a PR name (but that is ok because we are here to learn and it is totally fine to make mistakes. I've made some myself when I first joined 😄). If you would like more help with understanding Git feel free to reach out.